Readme.md

```diff
@@ -9,6 +13,7 @@ you can then enable the hook.
 
 ## Testing
 Clone the repository and execute ./test-hook.sh
+Also testIfWinValidFileName.sh may be tested in advance to git commit to check for Windows filename compatibility to avoid too much I/O.
```

Comment:
Can be removed as `testIfWinValidFileName.sh` will not be kept.

Reply:
Why shouldn't the file "testIfWinValidFileName.sh" be kept?
I think it is senseful to use a function that can be used by both the hook itself and to test externally.
Would be using a function "testIfWinValidFileName" inside an existing shell file be ok?

Reply:
Having too files makes it harder for users, and I think it should be easy to use.